components/Feature/ReferralsTable/index.module.scss

```diff
@@ -0,0 +1,15 @@
+.rejected {
+  background-color: #d4351c;
+  color: white;
+}
+.sent {
+  background-color: #1d70b8;
+  color: white;
+}
+
+.sent,
+.rejected,
+.accepted {
+  padding: 4px 4px 4px 4px;
+  border: solid 1px black;
```

Comment:
Pedantic comment of the week but I would personally remove this border. Just a design preference, wouldn't need to hold up a deployment :-)

Reply:
Should I still keep it for `accepted` as it doesn't have a background color 🤔 ?